__tests__/app.test.js

```diff
@@ -105,10 +105,56 @@ describe( "/api/articles", ()=> {
             expect(typeof article.votes).toBe("number")
             expect(typeof article.article_img_url).toBe("string")
             expect(typeof article.comment_count).toBe("number")
-            expect(Object.keys(article).includes('body')).toBe(false)
+            expect(article).not.toHaveProperty("body")
             })
         })
     })
 })
 
+describe( "/api/articles/:article_id/comments", ()=> {
+    it("GET 200 sends all available comments with properties for the given article_id, and most recent comments served first", () => {
+        
+        return request(app)
+        .get("/api/articles/9/comments")
+        .expect(200)
+        .then(({body}) => {
```

Comment:
We need a length check in this test to make sure we're not getting a false positive. If the array is empty, this will still pass